### ISSUE TITLE
It's time for new release

### ISSUE THREAD

**Issue by RajashekarRaju** [kelseyhightower/nocode#4714]

**Comment by RMcD** [comment]
Agreed, we need to handle for the harmonic frequency distortion that occurs when writing this application to a crystal lattice storage device that results in the lattice losing cohesion.
Also, there is some evidence this issue may extended to DNA based storage as the base pairs struggle to bond.
My proposed solution is to add isomer metadata to the header on a storage type basis, however it will need to default to not storing the document if the metadata is null or empty as we need to maintain this programs status as a reference implementation of the NESE (never ever store empty) -- nee-s-uh standard. :-)

**Comment by RajashekarRaju** [comment]
Believe me when I say, I really understood what you've said. But I would make one small change, I would prefer symphony over harmonic. No offense.
